### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by rn1357**
Image preprocessing before getting hashes

### CODE REVIEW

File: FingerPrintsEnhancement.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import glob
+input_path = r"/Users/rakshithanagendrappa/Desktop/Distorted/*.bmp"
+out_path = '/Users/rakshithanagendrappa/Desktop/DistortedEnhanced/'
+
+image_paths = list(glob.glob(input_path))
+for i, img in enumerate(image_paths):
+    img_name = img.split("/")[-1].split("_")[0]
+    image = cv2.imread(img)
+    out = fingerprint_enhancer. enhance_Fingerprint(image) 
+    f_image = cv2.resize(out, (448, 480))
```
Comment (josedeveloper2026) [inline FingerPrintsEnhancement.py:13]: where does the 448 and 480 come from?

File: FingerPrintsEnhancement.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import fingerprint_enhancer					# Load the library 
+import cv2 
+import numpy as np
+import glob
+input_path = r"/Users/rakshithanagendrappa/Desktop/Distorted/*.bmp"
+out_path = '/Users/rakshithanagendrappa/Desktop/DistortedEnhanced/'
+
+image_paths = list(glob.glob(input_path))
+for i, img in enumerate(image_paths):
+    img_name = img.split("/")[-1].split("_")[0]
```
Comment (josedeveloper2026) [inline FingerPrintsEnhancement.py:10]: what is the naming convention? Can you please document?

File: FingerPrintsEnhancement.py
Comment (josedeveloper2026) [inline FingerPrintsEnhancement.py:1]: Can you please make this also a function and parameterize? 
and then use

```
if __name__== ...
```
if there is a need to call and run this. Make the values of the input and output come from the command line parameters. In the simplest case, you can read it through `sys.argv` 

You have to build the entire scripts as a pipeline. A pipeline means there are several scripts to process separate parts of the tasks. And the output of one is the input of the next node. They should all be parameterized and parameters should come from command line. There will be a shell script in the end that connects all steps of pipelines. Does it make sense? Can you please organize your code as such?

File: FingerPrintsEnhancement.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    image = cv2.imread(img)
+    out = fingerprint_enhancer. enhance_Fingerprint(image)
```
Comment (josedeveloper2026) [inline FingerPrintsEnhancement.py:12]: please document what is an enhancer?

File: FingerprintsOrientationMap.py
Comment (josedeveloper2026) [inline FingerprintsOrientationMap.py:1]: This code is taken from this URL: 
https://github.com/rayronvictor/Fingerprint-Features-Extraction/blob/master/orientation.py

Can you please a comments above this line and specify the link above and mention the code is taken from that link?

File: FingerprintsOrientationMap.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      for j in range(int(h/block_size)):  
+         dangle = 2*angles.item(j, i) 
+         v = int(round(dangle * (255/float(360)))) 
+         for k in range(block_size): 
+            for l in range(block_size): 
+               im.itemset((j*block_size+l,i*block_size+k), v) 
+   return im 
+ 
+if __name__ == '__main__': 
+   input_path = r"/Users/rakshithanagendrappa/Desktop/Distorted/*.bmp"
```
Comment (josedeveloper2026) [inline FingerprintsOrientationMap.py:109]: again, please parameterize this and get it from command line.

File: FingerPrintsExtraction.py
Comment (josedeveloper2026) [inline FingerPrintsExtraction.py:1]: Same for this file, please specify where did you brow it from.

**Review by josedeveloper2026 — COMMENTED** [review]
please fix these as well.

File: FingerPrintsEnhancement.py (hunk 1 above)
Comment (rn1357) [reply FingerPrintsEnhancement.py:13]: I am resizing the image based on the input to be given, When I read the paper related to hash techniques, they said it needs to be resized before inputting to the values I have given.

File: FingerPrintsEnhancement.py (hunk 2 above)
Comment (rn1357) [reply FingerPrintsEnhancement.py:10]: Yes Dr. Hossein, I will do that.

File: FingerprintsOrientationMap.py
Comment (rn1357) [reply FingerprintsOrientationMap.py:1]: Sure!

File: FingerPrintsExtraction.py
Comment (rn1357) [reply FingerPrintsExtraction.py:1]: All the references of code and dataset, I have put in README file prof.

File: FingerPrintsEnhancement.py (hunk 1 above)
Comment (josedeveloper2026) [reply FingerPrintsEnhancement.py:13]: Can you please add some comment line above this code and provide good pointer why you are doing this?

Comment (josedeveloper2026) [reply FingerPrintsEnhancement.py:13]: also did they said what should be the size?

File: FingerPrintsExtraction.py
Comment (josedeveloper2026) [reply FingerPrintsExtraction.py:1]: Also please put them above each file or if only one function is taken exactly from somewhere also mention that.

File: FingerPrintsEnhancement.py (hunk 1 above)
Comment (rn1357) [reply FingerPrintsEnhancement.py:13]: They took 448 and 480, because it is easily divisible by 8

**Pull request closed without merging**